playground/publishers/Publishers.AppHost/api-identity.module.bicep

```diff
@@ -0,0 +1,15 @@
+@description('The location for the resource(s) to be deployed.')
```

Comment:
Maybe let's revert this app. It currently its set up to generate a manifest through `eng\refreshManifests.ps1` since it doesn't have a generate-manifests launch profile.

And all this bicep is duped in the `infra` folder.

Reply:
That explains the huge delta here. I originally thought we weren't proactive about updating the manifests here. Yet another case for https://github.com/dotnet/aspire/issues/8292.